### DISCUSSION TITLE
[Question] Rotate property and space

### DISCUSSION THREAD

**Discussion by ingdesarr1** [flet-dev/flet#2060] [Q&A] [answered]
### Question

Good afternon. I'm testing the rotate control to use with a Progressbar, to display it vertically. The problem is that when rotating the element, flet detects that it is still occupying the same space horizontally and not vertically. If I reduce the horizontal space, this reduces the element and the display in rotation is affected. How can I make the ProgressBar look vertically but without this problem? I try with a Container, but the problem still. Thanks

![image](https://github.com/flet-dev/flet/assets/146734555/f7d3ebbc-3639-4696-8f2e-7259691ce362)

### Code sample

```python
from time import sleep
import math

import flet as ft

def main(page: ft.Page):
    pb = ft.ProgressBar(height=20,width=100,rotate=ft.Rotate(-math.pi/2,alignment=ft.alignment.top_left),value=0)

    def cambio(e):
        pb.value = e.control.value/100
        page.update()

    page.add(
        ft.Slider(min=0,max=100,divisions=100,label="{value}%",on_change=cambio),
        ft.Text("Linear progress indicator", style="headlineSmall"),
        ft.Column([ ft.Text("Doing something..."), 
                   ft.Row(
                       controls=[
                        #    ft.Container(width=10,content=pb,bgcolor="red",rotate=ft.Rotate(-math.pi/2,alignment=ft.alignment.top_left)),
                            pb,
                           ft.Container(bgcolor="blue",width=50,height=50)
                       ]
                   )
                   ]),
    )

ft.app(target=main)
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment, marked as answer]
This is a bug.
There's an opened issue: https://github.com/flet-dev/flet/issues/1653

**Reply by ingdesarr1** [reply]
Thanks
